### PR TITLE
Bugfix/jcrvlt 683

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

**Comment by kwin** [comment]
@adamcin Please have a look.

**Review by adamcin — APPROVED** [review]

File: vault-core/src/main/java/org/apache/jackrabbit/vault/fs/io/Importer.java [hunk 1]
```diff
@@ -565,6 +574,52 @@ public void run(Archive archive, Session session, String parentPath)
...
+                                aclHandlingForRestoredPolicy = AccessControlHandling.IGNORE;
+                                break;
+                            case IGNORE:
+                                aclHandlingForRestoredPolicy = AccessControlHandling.OVERWRITE;
+                                break;
+                            case CLEAR:
+                                aclHandlingForRestoredPolicy = AccessControlHandling.IGNORE;
+                                break;
+                            case MERGE:
+                                aclHandlingForRestoredPolicy = AccessControlHandling.MERGE;
```
[the remark this answers is not in the document]
Comment (kwin) [reply vault-core/src/main/java/org/apache/jackrabbit/vault/fs/io/Importer.java:604]: @adamcin Can you come up with a dedicated JIRA issue and first a PR to fix the javadoc according to the status quo (for `AccessControlHandling`). Afterwards we can discuss modifying the behaviour.

**Comment by adamcin** [comment]
@kwin Please amend the commit with `--reset-author`. It currently looks like I've written all the changes in the PR.

**Comment by kwin** [comment]
> @kwin Please amend the commit with `--reset-author`. It currently looks like I've written all the changes in the PR.

Ok, I will make you co-author on the commit though (https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) as you provided the ITs.

**Comment by kwin** [comment]
Superseded by #297

**Pull request closed without merging**